package.json

```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@10.7.1 -W  && git checkout -- package.json",
```

Comment:
Is there a reason for using 10.7.1?

Reply:
CI will currently fail since we're pointing to a tagged release that's a minor version ahead if you don't do this. I'll revert before committing this, but need it for the CI branch to look good.